src/services/ProjectTokenHistoricalPriceService.ts

```diff
@@ -113,25 +113,25 @@ export class ProjectTokenHistoricalPriceService {
     const prices = await this.getFundingTokenUSDPricesByID(
       lbpMgr.fundingTokenInfo.id,
       endTimeSeconds,
-      startTimeSeconds,
+      startTimeSeconds < new Date().getTime() / 1000 ? startTimeSeconds : Math.floor(new Date().getTime() / 1000) - 3600,
```

Comment:
Generally is most efficient and most consistent in having a single sense of "now", to define `now` once at the top of a function instead of calling new Date over and over again, inefficiently and potentially giving you different values.